tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -24,6 +39,70 @@ DRAW_TEST_F(CGContext, RedBox, UIKitMimicTest) {
     CGContextFillRect(context, CGRectInset(bounds, 10, 10));
 }
 
+DRAW_TEST_F(CGContext, DrawIntoRect, UIKitMimicTest) {
+    // Create a context to draw the Image into
```

Comment:
i'll do that with the rebase, given most of the tests here aren't disabled as per now.

---
In reply to: [88964021](https://github.com/Microsoft/WinObjC/pull/1414#discussion_r88964021) [](ancestors = 88964021)